lib/pod_builder/command/install_sources.rb

```diff
@@ -21,6 +23,7 @@ def self.call
         
         framework_files.each do |path|
           rel_path = Pathname.new(path).relative_path_from(Pathname.new(base_path)).to_s
+          next if !OPTIONS.has_key?(:all) && !argument_pods.include?(rel_path.sub(/\/.*/m, ""))
```

Comment:
We could use the prebuilt name instead which looks more readable, what do you think?

```suggestion
          next if !OPTIONS.has_key?(:all) && !argument_pods.include?(File.basename(path, ".*"))
```

If it works it could be moved to line 25